djangocms_navigation/templatetags/navigation_admin_tree.py

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8 -*-
+from django import template
+from django.template.loader import get_template
+from django.utils.html import format_html
+from django.utils.safestring import mark_safe
+from django.templatetags.static import static
+
+from treebeard.templatetags import admin_tree
+
+register = template.Library()
+
+'''
+This module is simply for overwriting some of treebeard's admin_tree templatetag functions
+with djangocms_navigation specific values.
+'''
+
+
+def get_spacer(first, result):
```

Comment:
So is this how treebeard does this currently i.e. you're replacing kind of like for like template tags?

Reply:
I'm not a fan of having markup in template tags.

Reply:
Yes, totally agree. Treebeard encapsulates some markup in their template tags, so I'm overwriting specific snippets.